rcon/commands.py

```diff
@@ -366,12 +366,30 @@ def _read_list(self, raw: bytes, conn: HLLConnection) -> list[str]:
 
     @_auto_retry
     def _get_list(
-        self, item: str, can_fail=True, conn: HLLConnection | None = None
+        self,
+        item: str,
+        can_fail=True,
+        conn: HLLConnection | None = None,
+        fail_msgs: Sequence[str] = [],
```

Comment:
Is there any other known list command that returns other things than the actual list or FAIL? If not, what about having a more generic handling: when not fail and not a list, then it's a command failed error?

Reply:
There's also the possibility that there's still junk data on the buffer. While I doubt it'd have any noticeable impact, it shouldn't be handled the same way as a game server `FAIL`.